draft-ietf-quic-qpack.md

```diff
@@ -685,10 +713,34 @@ Header data is prefixed with two integers, `Largest Reference` and `Base Index`.
 
 `Largest Reference` identifies the largest absolute dynamic index referenced in
 the block.  Blocking decoders use the Largest Reference to determine when it is
-safe to process the rest of the block.
+safe to process the rest of the block.  If Largest Reference is greater than
+zero, the encoder transforms it as follows before encoding:
+
+~~~
+   LargestReference = LargestReference mod 2*MaxEntries + 1
+~~~
+
+The decoder reconstructs the Largest Reference using the following algorithm:
+
+~~~
+   if LargestReference > 0:
+      LargestReference -= 1
+      CurrentWrapped = TableLargestAbsoluteIndex mod 2*MaxEntries
+
+      if CurrentWrapped > LargestReference + MaxEntries:
```

Comment:
There's a couple bugs here.  It should be >= not >, and there is another case: the decoder's largest index has wrapped more times than LR.